apps/www/src/app/(public pages)/hiring-managers/learn/(pages)/about-digital-talent.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About the Digital Talent and Attraction Development Team

```

Comment:
This should read "About the Digital Talent Attraction and Development Team", as it is elsewhere in the document